# Topics Advertised
TrajectoryTopic:            '/example0/trajectory'
WayPointsTopic:             '/example0/waypoints'
RouteTopic:                 '/example0/route'
AccTopic:                   '/example0/acc'

# Frame Name
FrameName:                  'world'

# Params
WeightT:                     1024.0

WeightAcc:                   16.0

WeightJerk:                  0.4

MaxAccRate:                  1.50

MaxVelRate:                  0.80

Iterations:                  23

Epsilon:                     0.02

##process1
# Number of Points 
PointNumber:                   3

#process2
#Different Number of Points in every part .cpp文件中maxPointParts设置20,最多区间小于20即可
PointArray:                   [3,3,4,4]
#Maximum segment  此值小于等于PointArray元素个数即可
MaxParts:                       3

#process3
#The point coordinates to be passed in
                             #x       #y      #z
PointSet:                 [ 0.0,      0.0,    0.0,           
                            1,      2,      3,            
                            2,      3,      4,
                            3,      4,      5,
                            4,      5,      6]           
MaxPointSetNumber:              15         #3*3,PointSet元素个数，即坐标点数*3

#process4
#Set the coordinate points of multiple segments